tests/community_push/test_community_push.py

```diff
@@ -549,3 +550,109 @@ def test_beta_images_live_push(
+        "workflow": "community",
+        "mappings": {
+            "aws_storage": [
+                {
+                    "architecture": "x86_64",
+                    "destination": "fake-destination-access",
+                    "overwrite": False,
+                    "restrict_version": False,
+                    "provider": "awstest",
+                    "volume": "/dev/sda1",
+                    "meta": {
+                        "billing-code-config": starmap_ami_billing_config,
+                        "accounts": [
+                            "first_account",
+                            "second_account",
+                        ],
+                    },
+                }
+            ]
+        },
+    }
+    policy2 = {
+        "name": "test-product2",
+        "workflow": "community",
+        "mappings": {
+            "aws_storage": [
+                {
+                    "architecture": "x86_64",
+                    "destination": "fake-destination-access2",
+                    "overwrite": True,
+                    "restrict_version": True,
+                    "provider": "anotherprovider",
+                    "volume": "/dev/sda1",
+                    "meta": {
+                        "billing-code-config": starmap_ami_billing_config,
+                        "accounts": [
+                            "third_account",
+                            "fourth_account",
+                        ],
+                    },
+                }
+            ]
+        },
+    }
+    mock_starmap.query_image_by_name.side_effect = [
+        QueryResponse.from_json(pol) for pol in [policy1, policy2]
+    ]
+
+    # Test
+    command_tester.test(
+        lambda: entry_point(CommunityVMPush),
+        [
+            "test-push",
+            "--starmap-url",
+            "https://starmap-example.com",
+            "--credentials",
+            "eyJtYXJrZXRwbGFjZV9hY2NvdW50IjogInRlc3QtbmEiLCAiYXV0aCI6eyJmb28iOiJiYXIifQo=",
+            "--rhsm-url",
+            "https://rhsm.com/test/api/",
+            "--debug",
+            "koji:https://fakekoji.com?vmi_build=ami_build,ami_build_2",
+        ],
+    )
+
+    # Since the push item get updated with destination and some stuff it's easier to just
+    # change it to "mock.ANY" as we just want to test here whether the sharing accounts are correct
+    mock_cloud_instance.return_value.upload.assert_has_calls(
+        [
+            mock.call(
+                mock.ANY,
+                custom_tags=None,
+                container='redhat-cloudimg-fake-destination',
+                accounts=['first_account', 'second_account'],
+                snapshot_accounts=None,
+            ),
+            mock.call(
+                mock.ANY,
+                custom_tags=None,
+                container='redhat-cloudimg-fake-destination',
+                accounts=['third_account', 'fourth_account'],
+                snapshot_accounts=None,
+            ),
+        ]
+    )
```

Comment:
This test ensures we're calling each push item with the proper mapped sharing accounts, so we don't just rely on logs 😄